docs/smt_rmt_suma.md

```diff
@@ -1,51 +1,64 @@
-|Files deduplication|:heavy_check_mark:|:heavy_check_mark:|
-|Transfer registration data to SCC|:heavy_check_mark:|:heavy_check_mark:|
-|Reporting|:heavy_check_mark:|:x:|
-|Custom TLS certificates for web-server|:heavy_check_mark:|:heavy_check_mark:|
-|Clean up data from repositories that are not used any longer|:heavy_check_mark:|:heavy_check_mark:|
-|Bash completion|:x:|:heavy_check_mark:|
-|Available on [openSUSE Leap 15](https://github.com/SUSE/rmt/blob/master/docs/installation.md#installation-on-opensuse-leap-15)|:x:|:heavy_check_mark:<sup>[3](#self-support)</sup>|
-|Option to [run as container](https://github.com/SUSE/rmt/blob/master/README.md#development-setup---docker-compose)|:x:|:heavy_check_mark:<sup>[3](#self-support)</sup>|
-|Easy development setup + [contribution guide](https://github.com/SUSE/rmt/blob/master/docs/CONTRIBUTING.md)|:x:|:heavy_check_mark:|
-|100% test [coverage](https://coveralls.io/github/SUSE/rmt?branch=master)|:x:|:heavy_check_mark:|
-|[Plugin functionality](https://github.com/SUSE/rmt/blob/master/docs/PLUGINS.md)|:x:|:heavy_check_mark:|
-|Webserver|Apache2|Nginx|
-|Database|MariaDB|MariaDB|
-|Platform|Perl|Ruby|
+| Feature/Tech      | SMT           | RMT           | SUMA          |
+|-------------------|:-------------:|:-------------:|:-------------:|
+|Available on SLES 11|:heavy_check_mark:|:x:|:x:|
+|Available on SLES 12|:heavy_check_mark:|:x:|:x:|
+|Available on SLES 15|:x:|:heavy_check_mark:|:heavy_check_mark:|
+|Sync products data from SCC|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Mirror RPMs from repositories|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Select which products to mirror|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Serve RPMs via http/https|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Registration of SLE 15 systems|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Registration of SLE 12 systems|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Registration of SLE 11 systems|:heavy_check_mark:|:x:|:heavy_check_mark:|
+|Registration of RHEL 8 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of RHEL 7 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of RHEL 6 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of CentOS 8 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of CentOS 7 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of CentOS 6 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Oracle Linux 8 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Oracle Linux 7 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Oracle Linux 6 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of openSUSE Leap 15 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Ubuntu 20.04 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Ubuntu 18.04 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Ubuntu 16.04 systems|:x:|:x:|:heavy_check_mark:|
```

Comment:
I'd simplify this part of the table to "non-SLES products (RHEL, Ubuntu, etc)" or move openSUSE above and then say "non-SUSE products (RHEL, Ubuntu, etc)". The reason is if you include the full list of SUSE Manager clients, the table is huge: RHEL, Oracle Linux, CentOS, AlmaLinux, Rocky Linux, Ubuntu, Debian, Amazon Linux, Alibaba Cloud Linux, and we will keep adding :-)